fri/src/prover/tests.rs

```diff
@@ -22,7 +22,7 @@ fn fri_prove_verify() {
     let trace_length = 4096;
     let lde_blowup = 8;
 
-    let options = FriOptions::new(lde_blowup, 4, 256);
+    let options = FriOptions::new(lde_blowup, 2, 8);
```

Comment:
Maybe instead of replacing the previous test, we should add a new test? We could probably have a helper function which contains most of the code for the test, and then just invoke this function with different folding factors etc.

Reply:
Sounds like a great idea. I will do that!